src/player.js

```diff
@@ -94,7 +94,12 @@
       this._tickCurrentTime((this._timeline.currentTime - this._startTime) * this.playbackRate);
       scope.invalidateEffects();
     },
-    get playbackRate() { return this._playbackRate; },
+    get playbackRate() {
+      return this._playbackRate;
+    },
+    set playbackRate(value) {
+      this._playbackRate = value;
```

Comment:
The algorithm for setting the playback rate of a player also needs to adjust the current time to avoid a jump. See http://w3c.github.io/web-animations/#updating-the-playback-rate-of-a-player